Core/GameEngine/Include/GameNetwork/NetPacketStructs.h

```diff
@@ -352,17 +316,6 @@ struct NetPacketFileCommand {
 	NetPacketCommandIdField commandId;
 };
 
-// File announce command header (variable: filename and metadata follow)
-// Fixed fields: T + type, R + relay, P + playerID, C + commandID, D
-// Variable: null-terminated filename + UnsignedShort fileID + UnsignedByte playerMask
```

Comment:
Maybe preserve these fixed and variable fields comments that existed previously? The other structs have them too.